Makefile

```diff
@@ -27,9 +27,11 @@ help:
 	@echo "DOCKER DEVELOPMENT"
 	@echo "------------------"
 	@echo ""
-	@echo "  - \`make docker-build-cluster\`: build child_chain, watcher and watcher_info images \c"
+	@echo "  - \`make docker-build-start-cluster\`: build child_chain, watcher and watcher_info images \c"
```

Comment:
why this change?

Reply:
because it actually builds and start the cluster, not just build. just making it more obvious

Reply:
makes sense! cool!